charts/ai-proxy/values.yaml

```diff
@@ -0,0 +1,113 @@
+# Default values for ai-proxy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  repository: ghcr.io/pluralsh/ai-proxy
+  # This sets the pull policy for images.
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v1.0.0"
+
+config:
+  provider: openai
+  providerHost: changeme
```

Comment:
should this be a correct default?

Reply:
I guess I can use openai api as the default